Uzira.rte/Loadouts.ini

```diff
@@ -4,349 +4,168 @@
 
 AddLoadout = Loadout
```

Comment:
Please undo basically all of your uzira loadout changes. Remember, again, that loadouts are for humans, and if they suck, they're useless.

1. Uzira's actors all suck so they need swarms. Their loadouts should generally be as full as possible, and I deliberately went through and did my utmost to support that before.
Also, with non-returnable craft it's even worse to have these anemic little loadouts. Your turret loadout spends 30oz on the craft and 14oz on the actual "turret", that's super useless!!!
If I'm buying loadouts for Uzira, it's cause I don't wanna do a million clicks to buy a decent number of actors.

2. While the spades for the engineer loadout are fun, I tried to consistently give engineer loadouts a breaching tool, so players can use it to quickly get one. I guess, really, engineer should be split into 2 or 3 loadouts (digger, breacher, engineer for disarming mines and crap), but I don't really wanna bother with that, so I think it'd be good to put the remote explosives back. Uzira needs to breach doors too.

3. Please give Uzira back her medikits. She's supposed to be combat capable, and they provide really needed survivability, without the player having to go through the hassle of manually buying medikits for her. I can't stress enough that if loadouts suck, people will just not use them and they'll be useless, and that's very stupid.
Relatedly, please put back the medic loadout, for all the reasons I've harped on already. I made sure that the regular loadouts don't get medikits for Uzira, but getting rid of a simple medic loadout is silly. If you wanna give Uzira a faction-themed medikit or some other means of healing, I'm absolutely okay with that, but it's not reasonable to assume that people won't want to heal up their zombies or Uzira herself sometimes.

Reply:
This loadout change is my bad, it was sort of separate to everything else I was doing. Still, I'd much prefer giving Uzira a static health regen either instead of or along with perhaps one medikit. 

Also ever since I mentioned it in another comment, I checked, and the loadouts are indeed still being used by AI in Conquest (which I use a benchmark for "what the AI does"). I get what you're saying about user-friendliness though. Remember that the player can now order multiple deliveries in rapid succession by holding up or down.

Reply:
No argument against health regen for Uzira.

As mentioned in another comment, I don't see anything in MetaFight.lua suggesting the AI uses loadouts? Even in master branch.
And yeah, you can order multiple, but like I said - if 2/3 of your order price is a non-refundable craft, you're kinda getting ripped off eh?

Reply:
Coulda sworn `SetOverridePurchaseList` was used more often than it really is... whatever 

I'll remain sad about the thematical nature of factions, but it is what it is